src/features/init/services/init.service.ts

```diff
@@ -62,8 +63,9 @@ export class InitService extends AbstractService {
 
   private _configureApp(environment: Readonly<IEnvironment>): void {
     this._configureAppFromEnvironment(environment);
-    this._configureAppFromPackage();
-    this._configureAppFromGitHubReleases();
+    this._configureAppFromPackage().then((): void => {
+      this._configureAppFromGitHubReleases();
```

Comment:
Do that after reading the package to make sure that the app version was fetched before using it to get the release notes.